constants.js

```diff
@@ -71,7 +72,7 @@ const checksWhitelist = {
       wipCheck,
       jobCheck,
       modelCheck,
-      prTemplateCheck
+      prTemplateCheck,
```

Comment:
Unneeded change.

Reply:
Sorry for that, It was a mistake,Correcting it in the latest commit.